pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml

```diff
@@ -1,9 +1,10 @@
 %link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/dance-landing/dance-sponsor-logo.css'}
 
-%div.sponsor-logo-with-text
-  %p= I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_title)
-  %div
-    %img{src:"/images/fit-450/AFE-Dark-Orange-logo.png"}
+%div.sponsor-logo-with-text{style: "font-size: 24px; text-align: center; margin-top: 60px"}
```

Comment:
Nit: I like to keep my HAML as clean as possible. If there is an id or class, you can remove the %div part.

Also, these in-line styles can be moved to your css file.

Reply:
Same for line 6 and throughout the file.